.github/workflows/dataset-validation.yml

```
@@ -0,0 +1,262 @@
+              await github.rest.issues.createComment({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                body: summary
+              });
+            } catch (error) {
+              core.warning('Could not read validation summary: ' + error.message);
+            }
+
+      - name: Create validation failure issue
+        if: failure() && (steps.ucsc_validation.outcome == 'failure' || steps.atlas_validation.outcome == 'failure')
```

actionlint (1.7.8)

property "ucsc_validation" is not defined in object type {strategy: {conclusion: string; outcome: string; outputs: {string => string}}; validation: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)